### PR TITLE
certificates stores are now configurable

### PR DESCRIPTION
**Pull request by cristipogacean**

### CODE REVIEW

File: src/ModuleProcess.cs [hunk 1]
```diff
@@ -118,11 +119,16 @@ public class ModuleProcess : IProcessControl {
         private IContainer ConfigureContainer(IConfigurationRoot configuration) {
 
             var config = new Config(configuration);
+            var clientServicesConfiguration = new ClientServicesConfig(configuration);
```
Comment (marcschier) [inline src/ModuleProcess.cs]: Would suggest rolling the configuration interface into the Config class, same as the other config components.   Then you do not need to make changes in here, only in Config.cs

**Review by marcschier — CHANGES_REQUESTED** [review]

File: src/ModuleProcess.cs (hunk 1 above)
Comment (cristipogacean) [reply src/ModuleProcess.cs]: done

Comment (marcschier) [reply src/ModuleProcess.cs]: Ah - I am so sorry - I should have been clearer.  Create the ClientServicesConfig in the Config constructor and forward all to the inner object, overriding what you need to override.   But this is ok, we can change that in the future if we ever want to.

**Review by marcschier — APPROVED** [review]

File: src/ModuleProcess.cs (hunk 1 above)
Comment (cristipogacean) [reply src/ModuleProcess.cs]: got it, thanks for the explanation

**Pull request merged**